gcore/gdalallvalidmaskband.cpp

```diff
@@ -82,6 +82,20 @@ CPLErr GDALAllValidMaskBand::IRasterIO(GDALRWFlag eRWFlag, int, int, int, int,
     return CE_None;
 }
 
+/************************************************************************/
+/*                   EmitErrorMessageIfWriteNotSupported()              */
+/************************************************************************/
+
+bool GDALAllValidMaskBand::EmitErrorMessageIfWriteNotSupported(
+    const char *pszCaller)
+{
+    ReportError(CE_Failure, CPLE_NoWriteAccess,
+                "%s: attempt to write to a all-valid implicit mask band.",
```

Comment:
I'm non-native English speaker so I might be wrong but this sounds better to me:
```suggestion
                "%s: attempt to write to an all-valid implicit mask band.",
```

Reply:
fixed